README.md

```diff
@@ -383,6 +397,7 @@ Please [discover modm's peripheral drivers for your specific device][discover].
 <td align="center">✗</td>
 <td align="center">✗</td>
 <td align="center">✗</td>
+<td align="center">✗</td>
```

Comment:
V70 **does** have *Generic System Clock*, but it's controlled by PMC (power management controller - see datasheet, section 31.9). I don't think modm likes name clash like this, what should I do?

Reply:
Moreover, the clock implementation from SAMG seems to fit SAMV with small changes (didn't test it yet though) and SAMG also shows "X" in the *System Clock* row.

Reply:
Hm, my current approach doesn't scale well… Should throw out the heuristical regex approach and just define the correct mapping manually (as done already) and also search for a special comment inside the `module.lb` about compatibility?

```py
# driver = ["System Clock"], compat = ["gclk", "pmc"]
```

Or should it be part of the module description somehow?

Reply:
I mean, you shouldn't have to deal with this, I'll check this out next week and you can focus on supporting SAMV.